<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFERTResultCalculator</class>
 <widget class="QDialog" name="QFERTResultCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool: Result Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxA">
       <property name="title">
        <string> Parameter A </string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>file:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFRDRComboBox" name="cmbFileA"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>result group:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFResultsGroupComboBox" name="cmbResultGroupA"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>result:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QFRawDataResultsComboBox" name="cmbResultA"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labPropertiesA">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>properties:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxB">
       <property name="title">
        <string> Parameter B </string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>file:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFRDRComboBox" name="cmbFileB"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>result group:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFResultsGroupComboBox" name="cmbResultGroupB"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>result:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QFRawDataResultsComboBox" name="cmbResultB"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>properties:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labPropertiesB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSameFiles">
     <property name="text">
      <string>same files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSameResultGroups">
     <property name="text">
      <string>same result groups</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFMathParserWidget" name="widEquation" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>new result name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edtNewName"/>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&amp;Evaluate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/help.png</normaloff>:/lib/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/exit.png</normaloff>:/lib/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFRDRComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFResultsGroupComboBox</class>
   <extends>QComboBox</extends>
   <header>qfresultsgroupcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFRawDataResultsComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrawdataresultscombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFMathParserWidget</class>
   <extends>QWidget</extends>
   <header>qfmathparserwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkSameFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFERTResultCalculator</receiver>
   <slot>checkBoxesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSameResultGroups</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFERTResultCalculator</receiver>
   <slot>checkBoxesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QFERTResultCalculator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFERTResultCalculator</receiver>
   <slot>eval()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkBoxesChanged()</slot>
  <slot>eval()</slot>
 </slots>
</ui>
